HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -4,17 +4,17 @@ namespace HerPublicWebsite.BusinessLogic.Models;
 
 public record class Questionnaire
 {
-    public int QuestionnaireId { get; set; }
-
     public Country? Country { get; set; }
     public OwnershipStatus? OwnershipStatus { get; set; }
 
     public string AddressLine1 { get; set; }
     public string AddressLine2 { get; set; }
-    public string AddressLine3 { get; set; }
     public string AddressTown { get; set; }
+    public string AddressCounty { get; set; }
     public string AddressPostcode { get; set; }
-
+    
+    public int CustodianCode { get; set; }

```

Comment:
I may be misunderstanding how this all fits together, but are the people filling in the questionnaire going to know their custodian code? I assumed we'd be finding it from an API